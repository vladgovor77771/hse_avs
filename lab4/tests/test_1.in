4
qxyz
1
8
ouyt
1
4
mnkj
1
2
abcd
1
1
