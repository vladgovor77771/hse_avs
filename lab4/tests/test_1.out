abcd => bcde
mnkj => opml
ouyt => syCx
qxyz => yFGH
